frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/ServerEmbedCodePane.tsx

```diff
@@ -45,15 +48,19 @@ export const ServerEmbedCodePane = ({
   displayOptions,
   initialPreviewParameters,
   serverCodeOptions,
-  selectedServerCodeOptionName,
-  setSelectedServerCodeOptionName,
+  selectedServerCodeOptionId,
+  setSelectedServerCodeOptionId,
+  onCopy,
 
   className,
 }: EmbedCodePaneProps): JSX.Element | null => {
   const selectedServerCodeOption = serverCodeOptions.find(
-    ({ name }) => name === selectedServerCodeOptionName,
+    ({ id }) => id === selectedServerCodeOptionId,
   );
 
+  const canWhitelabel = useSelector(getCanWhitelabel);
+  const shouldShowDownloadData = canWhitelabel && resourceType === "question";
```

Comment:
@WiNloSt what's the connection between downloading data and whitelabeling? is it a proxy to check if we have a valid license?

Reply:
Yep, it's the original condition when we render the download option on embedding.

So the condition was actually if users have valid license then show this and also it needs to be questions.

Reply:
interesting, so we're using a unrelated feature toggle to detect if we're on pro/ee because we don't have a specific feature toggle for that?

Reply:
That was introduce will before we revamped the token features. Maybe we have more suitable selectors for that now. But I need to check.